tyk-single-dc/values.yaml

```diff
@@ -563,7 +563,9 @@ tyk-dashboard:
     tolerations: []
     affinity: {}
 
-    extraEnvs: []
+    extraEnvs:
+    - name: "TYK_DB_HOSTCONFIG_GATEWAYHOSTNAME"
+      value: *gwHostName
```

Comment:
can you please elaborate why do we need use anchor and set extra environment here?

Reply:
Value of `TYK_DB_HOSTCONFIG_GATEWAYHOSTNAME` need to be set based on the `gateway.hostname`.
Since dashboard component chart cannot access that value, I need to either move it to global section so that dashboard can access it or by using anchors.

Reply:
uh, i see. looks clean! thanks!